src/login.js

```diff
@@ -68,13 +68,14 @@ export default class Login extends React.Component {
           <div className="bottom-container">
             <div className="options-container">
               <div className="option-container">
+                <input type="checkbox" className="checkbox"></input>
                 <a className="option">Keep me logged in</a>
               </div>
               <div className="forgot-container">
-                <a className="forgot">Forgot <Link>username</Link></a>
+                <a className="forgot">Forgot <Link>Username</Link></a>
```

Comment:
Plz change  ```<a>``` to  ```<p>```. ```<a>``` is only used for hyperlinks to different pages. I am pretty sure the vertical spacing between labels would change as well so please change the padding or margin between the labels from our css file too!

Reply:
And the change should apply to the rest of ```a``` tags in this file too!